final_task/rss_reader/argparse_handler.py

```diff
@@ -125,12 +126,41 @@ def valid_date(date: str) -> datetime.datetime:
         raise custom_error.NotValidDateError(msg)
 
 
+def valid_directory_path(path: str, key: str) -> str:
+    """Checks if path is valid"""
+    if path[-1] != "\\":
+        path += "\\"
```

Comment:
This will break your application for Linux OS, check my comment on your app's work.